### PR TITLE
LPS-62457 Autogenerated

### PR DESCRIPTION
**Pull request by brandizzi**
cc @marcellustavares.

### CODE REVIEW

File: modules/apps/forms-and-workflow/calendar/calendar-service/src/main/java/com/liferay/calendar/service/impl/CalendarBookingServiceImpl.java [hunk 1]
```diff
@@ -427,13 +427,27 @@ public CalendarBooking invokeTransition(
...
+			calendarBookingId, status, 0, false, false, serviceContext);
+	}
+
 	@Override
 	public CalendarBooking invokeTransition(
-			long calendarBookingId, int status, long startTime,
+			long calendarBookingId, long startTime, int status,
 			boolean updateInstance, boolean allFollowing,
 			ServiceContext serviceContext)
 		throws PortalException {
```
Comment (brandizzi) [inline modules/apps/forms-and-workflow/calendar/calendar-service/src/main/java/com/liferay/calendar/service/impl/CalendarBookingServiceImpl.java]: Hey guys! @mbowerman @natocesarrego

I just changed the order of the arguments as requested by Brian Chan in https://github.com/brianchandotcom/liferay-portal/pull/44044#issuecomment-254886443. This is an interesting part of our code standards.

Thanks!

**Comment by pyoo47** [comment]
ci:retest

**Comment by brianchandotcom** [comment]
Merged. Thx.

**Comment by brianchandotcom** [comment]
@brandizzi please resend:

```

		int status = ParamUtil.getInteger(actionRequest, "status");

		long startTime = ParamUtil.getLong(actionRequest, "startTime");
```

inconsistent with

```
	public CalendarBooking invokeTransition(
			long calendarBookingId, long startTime, int status,
			boolean updateInstance, boolean allFollowing,
			ServiceContext serviceContext)
```

Please review all relevant vars for this pull and make them consistent. Thx.

**Pull request closed without merging**